src/components/questions/ImageSelector.tsx

```diff
@@ -64,6 +68,11 @@ const ImageSelector = (props: {
             }
           }
         }, 1000);
+        if (Platform.OS === 'android') {
```

Comment:
Denk dat t handiger is om die snackbars uit mijn connection PR te gebruiken. Die IOS alerts zijn redelijk hinderlijk voor je flow. Zou dan voor zowel iOS en Android die snackbar gebruiken, is t meteen gelijk ;)

Reply:
Methode gebruiken van feature/refactor